### PR TITLE
[TASK] Style cold wallet page

### PR DESCRIPTION
**Pull request by MatthewPhinney**

### CODE REVIEW

**Comment by MatthewPhinney** [comment]
Before:
screen shot 2015-10-27 at 12 23 57 pm
After:
screen shot 2015-10-27 at 12 21 36 pm

If this looks over-styled or visually confusing for any reason, let me know

**Comment by MatthewPhinney** [comment]
Updated and rebased:
screen shot 2015-10-27 at 1 38 13 pm

File: src/templates/tabs/coldwallet.jade [hunk 1]
```diff
@@ -6,27 +6,27 @@ section.single.ddpage.content(ng-controller="ColdWalletCtrl")
...
+              dt Network Fee
+              dd {{networkFee}} XRP
+      div.col-xs-12.col-sm-7
+        div.well
+          p(ng-hide="transactionError") {{lastTxn}} The transaction was validated at {{txnTime | date:'yyyy-MM-dd HH:mm:ss Z'}}.
+          p.error(ng-show="transactionError") {{transactionErrorMessage}}
+    div.row.row-content
+      div.col-xs-12
+        h4
+          a(href="https://ripple.com/build/gateway-guide/" target="_blank" style="text-decoration:none;color:black;") Account Flags
```
Comment (mrajvanshy) [inline src/templates/tabs/coldwallet.jade]: All styling code should go into the less files

**Comment by MatthewPhinney** [comment]
Changed it to use a font awesome icon.  I think it's a bit more clear that way.

**Comment by mrajvanshy** [comment]
LGTM

**Pull request merged**